src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
```

Comment:
add box-sizing: border-box to * selector

Reply:
Still not fixed

Reply:
Not fixed

Reply:
not fixed
move box-sizing: border-box to * selector, and remove from other selectors
```
* {
box-sizing: border-box;
}
```